src/sales/cart-persistent-configuration.md

```diff
@@ -47,4 +47,25 @@ If the session cookie expires while the customer is logged in, the persistent co
    Yes|If the session cookie expires, the persistent cookie is preserved. If a guest shopper later logs in or creates a new account, the shopping cart is restored.
    No|The shopping cart is not preserved for guests after the session cookie expires.
 
+1. **<span class="ee-only">Persist Wish List</span>** determines if the state of customer wish lists is retained when the session ends. Options:
```

Comment:
For list items, span is not needed and simple liquid tag is best

```suggestion
1. {:.ee-only}Set **Persist Wish List** to determine if the state of customer wish lists is retained when the session ends:
```

Reply:
done